projects/ccf-shared/src/lib/components/info/info-dialog/info-dialog.component.html

```diff
@@ -13,11 +13,17 @@ <h2 class="title" mat-dialog-title>{{infoTitle}}</h2>
               <h2 class="no-header-margin">{{content.title}}</h2>
             </mat-expansion-panel-header>
               <div class="top-padding">
-                <markdown class="variable-binding" [data]="content.content"></markdown>
+                <markdown class="variable-binding" [class.first]="i === 0" [data]="content.content"></markdown>
+                <div class="video-player" *ngIf="i === 0 && pageType === 'RUI'">
+                  <youtube-player videoId="142hGer4xvU"></youtube-player> 
+                </div>
+                <div class="video-player" *ngIf="i === 0 && pageType === 'EUI'">
```

Comment:
instead of adding a pageType how about videoId? No need for this sort of hard coding.